Data/SkillStatMap.lua

```diff
@@ -1513,5 +1513,9 @@ return {
 },
 ["support_added_cooldown_count_if_not_instant"] = {
 	mod("AdditionalCooldownUses", "BASE", nil)
+},
+["kill_enemy_on_hit_if_under_10%_life"] = {
+	mod("CullPercent", "MAX", nil), 
+	value = 10
 }
```

Comment:
```suggestion
},
```
Please add a trailing comma for smaller Git diffs in future PRs here.